.gitignore

```diff
@@ -6,4 +6,10 @@ crosstool-ng
 aesd-build
 .vscode/
 test.sh.log
+<<<<<<< HEAD
```

Comment:
Fix the merge markers here (you can just delete them in this case since you want to take both changes from upstream and your changes)